src/azul/logging.py

```diff
@@ -163,20 +172,51 @@ def silenced_es_logger():
             assert es_log.level == original_log_level
 
 
-def http_body_log_message(body_type: str,
-                          body: bytes | bytearray | str | IO[bytes] | IO[str] | None,
+type BodyType = Literal['request', 'response']
+type PrintableBody = bytes | bytearray | str | Mapping
```

Comment:
PL

Reply:
Types var was dropped and the types were inlined.